src/vivarium_public_health/plugins/parser.py

```diff
@@ -34,34 +35,32 @@
 
 
 class CausesParsingErrors(ParsingError):
-    """
-    Error raised when there are any errors parsing a cause model configuration.
-    """
+    """Error raised when there are any errors parsing a cause model configuration."""
 
     def __init__(self, messages: List[str]):
         super().__init__("\n - " + "\n - ".join(messages))
 
 
 class CausesConfigurationParser(ComponentConfigurationParser):
-    """
+    """Configuration parser for cause model configurations.
+
     Component configuration parser that acts the same as the standard vivarium
     `ComponentConfigurationParser` but adds the additional ability to parse a
     configuration to create `DiseaseModel` components. These DiseaseModel
     configurations can either be specified directly in the configuration in a
     `causes` key or in external configuration files that are specified in the
     `external_configuration` key.
+
     """
 
     DEFAULT_MODEL_CONFIG = {
         "model_type": f"{DiseaseModel.__module__}.{DiseaseModel.__name__}",
         "initial_state": None,
     }
-    """
-    If a cause model configuration does not specify a model type or initial 
+    """If a cause model configuration does not specify a model type or initial 
```

Comment:
My understanding is that the rules are less rigid for docstrings for variables, attributes, etc. I initially did it but didn't love how it looked. I'll modify it since you even asked, though.